client/src/components/FeedbackForm.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jsx-a11y/label-has-for */
+import React, { useState } from "react";
+import "./feedbackForm.css";
+
+const FeedbackForm = ({ student }) => {
+	console.log("student",student);
+	const [feedbackMessage, setFeedbackMessage] = useState("");
+	const [givenFeedback, setGivenFeedback] = useState([]);
+	const [feedbackList, setFeedbackList] = useState([]);
+
+	const handlerSubmit = (e) => {
```

Comment:
I would call this submitHandler, or handleSubmit

Reply:
Maybe to keep the name consistent with the one below "handleChange" make it handleSubmit.